checkov/terraform/checks/graph_checks/azure/AzureAutomationAccOverlyPermissive.yaml

```diff
@@ -0,0 +1,18 @@
+metadata: 
+  id: "CKV2_AZURE_24"
+  name: "Ensure Azure automation account is NOT overly permissive"
```

Comment:
it feels like the name doesn't fir the actual check logic, so I'm not sure what exactly is the goal